src/woo_publications/publications/models.py

```diff
@@ -191,6 +192,41 @@ class Publication(ModelOwnerMixin, models.Model):
         ),
     )
 
+    # Retention fields:
+    bron_bewaartermijn = models.CharField(
+        _("retention policy source"),
+        help_text=_(
+            "The source of the retention policy (example: Selectielijst gemeenten 2020)."
+        ),
+        max_length=255,
+        blank=False,
+        default="",
+    )
+    selectiecategorie = models.CharField(
+        _("selection category"),
+        help_text=_(
+            "The category as specified in the provided retention policy source (example: 20.1.2)."
+        ),
+        max_length=255,
+        blank=True,
+    )
+    archiefnominatie = models.CharField(
+        _("archive action"),
+        help_text=_("Determines if the archived data will be retained or destroyed."),
+        choices=ArchiveNominationChoices.choices,
+        max_length=50,
+        blank=False,
+        default=ArchiveNominationChoices.retain,
+    )
```

Comment:
I'm not sure if setting a default here makes sense - it needs to be derived from the relation information category (or categories). And as long as no information category is related, we cannot determine this at all. So, I think this must be `blank=True`. I'm not sure about the implications though.